ntbs-service/Pages/Search/Index.cshtml

```diff
@@ -193,74 +176,82 @@
+                    <select nhs-select-type="Standard" asp-for="SearchParameters.TBServiceCode" id="tb-service-dropdown"
+                                asp-items="Model.TbServices" classes="search-dropdown">
+                        <option value="" selected>Select TB Service</option>
+                    </select>
+                </div>
+
+                <div class="nhs-form-group--search">
+                    <label nhs-label-type="Standard" classes="search-label--long" for="country-dropdown"> Country of birth </label>
+                    <select nhs-select-type="Standard" asp-for="SearchParameters.CountryId" id="country-dropdown"
+                                asp-items="Model.Countries"  classes="search-dropdown">
+                        <option value="" selected>Select country</option>
+                    </select>
+                </div>
+            </div>
+            </nhs-details>
+
+            <div class="submit-search-button-container">
+                <input type="submit" value="Search" class="nhsuk-button submit-search-button" id="search-button" formaction="#search-results"/>
+            </div>
+        </form>
+    </div>
+    <br/>
+</nhs-width-container>
 @if (Model.SearchParameters?.SearchParamsExist == true)
 {
-    <div>
-        <div class="clear-search-button-container">
-            <a href="/search" class="clear-search-button">
-                Clear search & start again
-            </a>
-        </div>
-        <div>
-            @{
-                var resultCount = Model.SearchResults?.NumberOfResults ?? 0;
-                var resultsFoundString = $"{resultCount.ToString()} Result{(resultCount == 1 ? "" : "s")} found.";
-            }
-            <p> @resultsFoundString </p>
-            <p> To filter results, add more search terms and search again. </p>
-        </div>
-        @if (Model.SearchResults != null)
-        {
-            <ol class="search-results-list">
-                @foreach (var result in @Model.SearchResults)
-                {
-                    <li>
-                        <!-- Each search hit is displayed using a notification banner -->
-                        <partial name="../Shared/_NotificationBanner.cshtml" model="result" />
-                    </li>
+    <div class="search-results-section">
+        <nhs-width-container container-width="Standard">
+            <div class="search-results-summary">
+                @{
+                    var resultCount = Model.SearchResults?.NumberOfResults ?? 0;
+                    var resultsFoundString = $"{resultCount.ToString()} Result{(resultCount == 1 ? "" : "s")} found.";
                 }
-            </ol>
-        }
+                <p> @resultsFoundString </p>
+                <a href="/search" class="clear-search-button">
+                    Clear search & start again
+                </a>
+            </div>
+            <p> @("To filter results, add more search terms and search again.") </p>

```

Comment:
Is this wrapped in an `@()` because vs code was complaining about razor again?

Reply:
yep :(